### PR TITLE
zellij: add head build support

### PR DESCRIPTION
**Pull request by LukeCarrier**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/z/zellij.rb [hunk 1]
```diff
@@ -32,6 +33,6 @@ def install
 
   test do
     assert_match("keybinds", shell_output("#{bin}/zellij setup --dump-config"))
-    assert_match("zellij #{version}", shell_output("#{bin}/zellij --version"))
+    assert_match("zellij #{version}", shell_output("#{bin}/zellij --version")) unless build.head?
```
Comment (chenrui333) [inline Formula/z/zellij.rb:36]:
```suggestion
    assert_match("zellij #{version}", shell_output("#{bin}/zellij --version"))
```

Comment (chenrui333) [reply Formula/z/zellij.rb:36]: we dont test head

**Comment by chenrui333** [comment]
@LukeCarrier can you amend commit msg as `zellij: add head build support`?

**Comment by LukeCarrier** [comment]
> @LukeCarrier can you amend commit msg as `zellij: add head build support`?

Done, but this makes the commit style check fail. Maybe it doesn't like the colon, or the lack of a description? Formula name and line length should match.

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**